app/assets/stylesheets/forever_style_guide/modules/_nav-icons.scss

```diff
@@ -140,24 +146,23 @@ a.cart-icon {
   min-width: $cart-count-width;
   padding-top: 1px; // vertical align number in circle
```

Comment:
```
@media (min-width: $grid-float-breakpoint-max) and (max-width: $screen-lg) {
    padding-top: 0;
}
```
Not sure if this is the best fix, but the positioning of the cart count seems off between those breakpoints
![screen shot 2017-02-07 at 3 55 20 pm](https://cloud.githubusercontent.com/assets/1926679/22711085/df1ea50c-ed4d-11e6-98c4-da3ceaeeb6b6.png)  ![screen shot 2017-02-07 at 3 56 22 pm](https://cloud.githubusercontent.com/assets/1926679/22711122/0a009d5c-ed4e-11e6-848e-a2ea39b20f23.png)